src/components/views/elements/AppTile.tsx

```diff
@@ -406,6 +409,11 @@ export default class AppTile extends React.Component<IProps, IState> {
             ? Container.Right
             : Container.Center;
         WidgetLayoutStore.instance.moveToContainer(this.props.room, this.props.app, targetContainer);
+        if (targetContainer === Container.Right
+            && RightPanelStore.getSharedInstance().visibleRoomPanelPhase === RightPanelPhases.TimelineCard) {
```

Comment:
I decided to not include that in this PR. The things not included are listed in the PR description.
> This does not include:
>  - only showing the rightPanelTimeline in rooms that have a maximised widget

We can however just hide them for now. But this will be overhauled when working on https://github.com/vector-im/element-web/issues/19632 anyways.

Reply:
Hmm, after testing it again today, I think we should at least change the right panel phase when entering a room without maximised widgets as an initial approach. It's quite confusing / disturbing to see two timelines on screen together, so let's ensure that is not possible.

Reply:
I came to the conclusion that as well. Sadly took me a while today (I wanted it to be a little change since it needs to be changed again when we overhaul the right-panel navigation)... but now its here: [`f8ae27e` (#7112)](https://github.com/matrix-org/matrix-react-sdk/pull/7112/commits/f8ae27ee1857ec5bfddf3dd5812f72b9a8391178) now.